auth/host_session.go

```diff
@@ -237,16 +237,17 @@ func CheckNetRegAndHostUpdate(networks []string, h *models.Host) {
 				Host:   *h,
 				Node:   *newNode,
 			})
+			if servercfg.IsMessageQueueBackend() {
```

Comment:
Why do we have this check? Shouldn't we always assume it's a message queue? Otherwise just wouldn't work, right?

Reply:
this check is at multiple places, would be better to get rid of it in a separate PR.
I believe we have a ticket already to do this.